index.js

```diff
@@ -81,15 +55,30 @@ function processArgv(argv) {
       break;
   }
 
-  // CLI handles new command without passing through to modules
-  if (passToModules) {
-    if (fs.existsSync(packageJsonPath)) {
-      const modules = getModules(require(packageJsonPath));
-      runCommand(modules, command, argv);
-    } else {
-      logger.info('No package.json file found in current working directory.');
+  // Look globally and locally for matching glob pattern
+  const dirs = [
+    `${process.cwd()}`, // local (where they ran the command from)
+    `${__dirname}/..`   // global (where this code exists)
+  ];
+
+  getGlobs(dirs).forEach(pkg => {
+    let module
```

Comment:
Did the linter not pick the missing semicolon up?

Reply:
Great catch @blackbaud-stevebrush.  I wasn't linting the `index.js` file.  I've enabled that and fixed a few other errors in addition to the one you found.